pontoon/machinery/utils.py

```diff
@@ -61,6 +79,45 @@ def get_google_translate_data(text, locale_code):
         }
 
 
+def get_google_automl_translation(text, locale):
+    try:
+        client = translate.TranslationServiceClient()
+    except DefaultCredentialsError as e:
+        log.error("Google AutoML Translation error: {e}")
+        return {
+            "status": False,
+            "message": f"{e}",
+        }
+
+    project_id = "85591518533"
+    model_id = locale.google_automl_model
+    location = "us-central1"
```

Comment:
This also looks like a magic constant. At the very least it should have a code comment explaining itself.

Reply:
I'll add a comment here - AutoML requires `us-central1`.

Although the docs are not [explicit](https://cloud.google.com/translate/automl/docs/before-you-begin#create_a_cloud_storage_bucket) about it, I tried other locations and got an error stating that.